src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -243,6 +244,21 @@ private static boolean isPresent(String availableChecks, String checkName) {
         return availableChecks.matches(linkPattern);
     }
 
+    @Test
+    public void testAllConfigsHaveLinkInSite() throws Exception {
+        final String siteContent = Files.readString(SITE_PATH);
+
+        for (Path path : XdocUtil.getXdocsConfigFilePaths(XdocUtil.getXdocsFilePaths())) {
```

Comment:
I am bit confused.
getXdocsConfigFilePaths find all files by prefix config
https://github.com/checkstyle/checkstyle/blob/c3ab3fd9eb4dddfaba54921a192a0be9f0a06f44/src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/XdocUtil.java#L79
 
But we do not have such files after split https://github.com/checkstyle/checkstyle/tree/master/src/xdocs

Reply:
We have an issue for that. I have to do it:

https://github.com/checkstyle/checkstyle/issues/13132

Reply:
Ok, but what is reason of this PR ? 
Should we combine two issues?

Reply:
This PR came from  https://github.com/checkstyle/checkstyle/pull/13295#pullrequestreview-1496521604

I will do #13132 in another PR

Reply:
Ok, please confirm that this PR is not doing any valuable validation.

Ad if so, can we close it and simply wait for 13123.

Reply:
If there is some benefit please share.

Reply:
Example of failure is here: https://github.com/checkstyle/checkstyle/pull/13327#issuecomment-1613267207

Reply:
According to code it should not do violation, as Check is not file "config_". 
So I am confused.

Reply:
ok, I misread this getXdocsConfigFilePaths method.
it works
![image](https://github.com/checkstyle/checkstyle/assets/812984/ab582a93-f20d-474e-b541-5199a134cfe5)